modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/UrlUtil.kt

```diff
@@ -21,4 +23,23 @@ class UrlUtil @Inject constructor() {
         }
         bytes
     }
+
+    @Throws(IOException::class)
+    suspend fun contentString(url: String) = withContext(Dispatchers.IO) {
+        var inputStream: InputStream? = null
+        val sb = StringBuilder()
+        var line: String?
+
+        try {
+            inputStream = URL(url).openStream()
+            val reader = BufferedReader(InputStreamReader(inputStream))
+            while ((reader.readLine().also { line = it }) != null) {
+                sb.append(line).append(System.lineSeparator())
+            }
+        } finally {
+            inputStream?.close()
+        }
+
+        sb.toString()
```

Comment:
This can be replaced with `URL(url).readText()`. Or if you need a line separator after last line.

```
buildString {
  append(URL(url).readText())
  append(System.lineSeparator())
}
```

Reply:
Nice! In 381e221